.github/workflows/daily_ete_test.yml

```diff
@@ -1,4 +1,4 @@
-name: test
+name: daily_ete_test
```

Comment:
do you have a ran log on your forked repo?

Reply:
> ran

I‘m running on it, now it takes too long to run one round. I will attach it.